components/server/src/express-util.ts

```diff
@@ -33,6 +33,9 @@ export const isAllowedWebsocketDomain = (originHeader: string, gitpodHostName: s
         if (originHostname === gitpodHostName) {
             return true;
         }
+        if (strict) {
+            return false;
+        }
         if (looksLikeWorkspaceHostname(originUrl, gitpodHostName)) {
```

Comment:
Is it safe to delete this method?

Reply:
No, /api/v1 should use it still, but it is protected with bearer token